cmd/influx_tools/internal/format/binary/binary.pb.go

```diff
@@ -45,7 +24,7 @@ var _ = time.Kitchen
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
We are in fact using the newest version of `github.com/gogo/protobuf/proto`, so this is safe.